util/Migrator/DbScripts/2020-06-16_00_OrgSso.sql

```diff
@@ -10,3 +10,59 @@ BEGIN
     );
 END
 GO
+
+IF EXISTS(SELECT * FROM sys.views WHERE [Name] = 'SsoConfig')
+    BEGIN
```

Comment:
small nit, do not indent the `BEGIN` and `END` for `IF` blocks in T-SQL for these scripts, the `DROP`, etc. would still be indented, but only once.

e.g.
```sql
IF EXISTS (BLAH BLAH BLAH)
BEGIN
    -- DO STUFF!
END
```

Reply:
nit away, gotta make sure my formatter is setup correctly.